Source/Scene/FXAA.js

```diff
@@ -88,7 +88,7 @@ define([
                 pixelDatatype : PixelDatatype.UNSIGNED_BYTE
             });
 
-            if (context.depthStencilTexture) {
+            if (context.depthTexture) {
```

Comment:
Not related - but a mistake from a search and replace I did in https://github.com/AnalyticalGraphicsInc/cesium/pull/4747